src/sbvr-api/permissions.ts

```diff
@@ -684,6 +684,161 @@ const getAlias = (name: string) => {
 	}
 	return `permissions${permissionsJSON}`;
 };
+
+const rewriteRelationship = memoizeWeak(
+	(
+		value: Relationship,
+		name: string,
+		abstractSqlModel: AbstractSqlModel,
+		permissionsLookup: PermissionLookup,
+		vocabulary: string,
+	) => {
+		let escapedName = sqlNameToODataName(name);
+		if (abstractSqlModel.tables[name]) {
+			escapedName = sqlNameToODataName(abstractSqlModel.tables[name].name);
+		}
+
+		const originalAbstractSQLModel = sbvrUtils.getAbstractSqlModel({
+			vocabulary,
+		});
+
+		const rewrite = (object: AnyObject) => {
+			if (_.has(object, '$') && _.isArray(object['$'])) {
```

Comment:
What is the reason for the `_.has` vs other checks?

Reply:
I think this was an artifact from a previous version